cirq/circuits/circuit_test.py

```diff
@@ -336,19 +334,19 @@ def test_repr(circuit_cls):
 ])"""
     )
 
-    c = circuit_cls(device=cg.Foxtail)
+    c = circuit_cls(device=foxy)
     cirq.testing.assert_equivalent_repr(c)
-    assert repr(c) == f'cirq.{circuit_cls.__name__}(device=cirq.google.Foxtail)'
+    assert repr(c) == f'cirq.{circuit_cls.__name__}(device=cirq.circuits.circuit_test.foxy)'
```

Comment:
Can this just be `device=foxy` instead? Same below.

Reply:
yes!

Reply:
Oh, actually, it cannot, as this is then used above in the assert_equivalent_repr - so the testclass have to generate repr that actually works.